content/tutorials/sample-tutorial.md

```diff
@@ -0,0 +1,30 @@
+# Sample Tutorial
+
+## Metadata
+[Metadata overview="Learn how to create and style custom widgets in Dojo.", topic="widgets"]
+
+## Aside
+[Aside title="Mandatory object for properties"]
+The 2nd argument of the `w()` function is mandatory even you have no properties to pass in. This is to ensure the correct type guarding for all widgets in TypeScript.
+[/Aside]
+
+## Task
+<dojo-task>Create a new root node for the application</dojo-task>
```

Comment:
Shouldn't this be using the `[Task....]....[/Task]` pragma over a custom element?

Reply:
Missed this one. Needed to change that back.